src/common/common-md-flag/common-md-flag.vcxproj

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Maybe we can use a better name for this? Since it's specifically for using common with interop something related to that like common-interop or something along those lines?